apps/worker/src/app/workflow/services/standard.worker.ts

```diff
@@ -57,22 +59,26 @@ export class StandardWorker extends StandardWorkerService {
     };
   }
 
-  private extractMinimalJobData(job: any): {
+  private extractMinimalJobData(data: IStandardDataDto): {
     environmentId: string;
-    organizationId: string;
     jobId: string;
+    organizationId: string;
     userId: string;
   } {
-    const { _environmentId: environmentId, _id: jobId, _organizationId: organizationId, _userId: userId } = job;
+    const { _environmentId: environmentId, _id: jobId, _organizationId: organizationId, _userId: userId } = data;
 
     if (!environmentId || !jobId || !organizationId || !userId) {
-      const message = job.payload.message;
+      const message = data.payload?.message;
```

Comment:
This object is structure data.payload?.message deprecated, i could not locate any place in the code that passes this data.

Reply:
Hard to say, I am okay with keeping it like that for now, but create a ticket to remove this redundant code so it won't stick with us and confuse the developers in the future.

* i will create a ticket rn